### PR TITLE
Simplify iconify_auto_link

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

**Comment by jcoyne** [comment]
@awead what do you think of this? Am I missing anything?

File: app/helpers/sufia/sufia_helper_behavior.rb [hunk 1]
```diff
@@ -132,7 +132,7 @@ def index_field_link(options)
     # @return [ActiveSupport::SafeBuffer]
     # @todo stop being a helper_method, start being part of the Blacklight render stack?
     def iconify_auto_link(text, show_link = true)
-      text = index_presenter(text[:document]).field_value(text[:field].to_sym, text[:config]) if text.is_a? Hash
+      text = text[:value] if text.is_a? Hash
```
Comment (cbeer) [inline app/helpers/sufia/sufia_helper_behavior.rb]: Do we need to worry about multivalued fields?

**Comment by atz** [comment]
There are some pending tests that now might pass if this works.  Or if they don't, should be revisited for whether they are considered correct.  Basically iconify was not testable for legitimate Hash args.

**Comment by awead** [comment]
@jcoyne you know, that's what I initially tried but I thought to myself, that can't be right... there has to be some Blacklight voodoo magick that I'm skipping. So yeah, 👍. You can do a join for multivalued fields.

**Comment by jcoyne** [comment]
@awead I've updated to handle arrays. It appears that's whats coming through with these `_tesim` fields.

**Comment by awead** [comment]
@jcoyne looks good. Should there be a separator for the join? Just a space?

**Comment by jcoyne** [comment]
@awead sure, separator added

**Pull request merged**